valgrind.supp

```diff
@@ -721,3 +721,13871 @@
    fun:wait_request
    fun:main
 }
+{
```

Comment:
changes to this file don't look correct to me. What I will suggest is take valgrind supp file from https://github.com/python/cpython/blob/3.6/Misc/valgrind-python.supp and add only PBS entries from old valgrind.supp file.